cmd/multiFaSequenceSwap/multiFaBedSequenceSwap.go

```diff
@@ -0,0 +1,89 @@
+package main
```

Comment:
You'll notice the gitactions failed for this PR. This is because gonomics expects the directory name for commands to be equal to the command file name. In this case, there is a 'Bed' in the filename, but not in the directory name. The right fix here is to rename these files to toss the 'Bed' part. Then we can recommit and see if it passes tests